src/workflow/confirm_time.c

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "confirm_time.h"
+#include <time.h>
+#include <workflow/confirm.h>
+
+bool workflow_confirm_time(uint32_t timestamp, int32_t timezone_offset)
```

Comment:
this should be named `workflow_confirm_date`, as the time is not confirmed.

For the backup, this is okay, but for U2F, you could run into issues if you restore on the same day as when you register on a u2f site, but the host machine time is too early (but same day) :nerd_face: 

Not sure if this edge case is worth handling :man_shrugging: I guess not (and the edge case is fixable by re-doing it a day later).

Reply:
Maybe we should make this case / assumption explicit in comments in the code, so it's clear this was deliberate and not an accident.

Reply:
I added support for checking the actual time. Should we do that for u2f?

Reply:
Not sure tbh. I'd be a bit worried that for some people the time might be off by a few hours, which would cause some anxiety when trying to restore. I did have this problem in a previous project.

Maybe let's go with time anyway and see what happens, as it would be more correct.

Reply:
ack